lib/utils.js

```diff
@@ -3,6 +3,7 @@
 const dateformat = require('dateformat')
 const stringifySafe = require('fast-safe-stringify')
 const defaultColorizer = require('./colors')()
+const chalk = require('chalk')
```

Comment:
As Matteo said, we have a standardized colorizer in this module. It is defined on line 5 immediately before this new import. Please use that colorizer to add color to output.